src/emm/emm_cli.py

```diff
@@ -170,7 +172,9 @@ def cli(ctx: click.Context, modules_dir: str, evg_config_file: str, evg_project:
     def dependencies(binder: inject.Binder) -> None:
         binder.bind(EvergreenApi, evg_api)
         binder.bind(EmmOptions, ctx.obj)
-        binder.bind(GithubService, GithubService.create())
+        binder.bind_to_constructor(GithubService, GithubService.create)
```

Comment:
I changed these so that they would get lazily initialized, otherwise a missing command would raise an exception before we get to the validation code and nice error messages.